### PR TITLE
Fixed to RAILS_MASTER_KEY as a default env key for decrypting.

### PR DESCRIPTION
**Pull request by morgoth**
Fixes mistake left in https://github.com/rails/rails/pull/33521/files#diff-2a29095afcfe2c683b82a779a94c2208R59
and misunderstanding in https://github.com/rails/rails/commit/d69b04de0ff33237209afea6f6cac3ab27934908

r? @y-yagi

### CODE REVIEW

File: railties/lib/rails/commands/credentials/credentials_command.rb [hunk 1]
```diff
@@ -55,10 +55,6 @@ def key_path
           options[:environment] ? "config/credentials/#{options[:environment]}.key" : "config/master.key"
         end
 
-        def env_key
-          options[:environment] ? "RAILS_#{options[:environment].upcase}_KEY" : "RAILS_MASTER_KEY"
-        end
-
```
Comment (kamipo) [inline railties/lib/rails/commands/credentials/credentials_command.rb]: This double blank lines are intended?

Comment (morgoth) [reply railties/lib/rails/commands/credentials/credentials_command.rb]: Yes, similar to what is in https://github.com/rails/rails/blob/b4b70ef2bfd922723aa02578d3feba887f5249dc/railties/lib/rails/commands/credentials/credentials_command.rb#L73-L74
I guess it was called "separation of concerns" in some commit, but I can't find it :-)

**Comment by y-yagi** [comment]
Oh, I misunderstood the conclusion of the discussion. 
(I thought it was implemented as per the https://github.com/rails/rails/pull/33521#issuecomment-412382031)

**Comment by y-yagi** [comment]
@morgoth  Thanks!

**Pull request merged**